Readme.md

```diff
@@ -34,4 +34,19 @@ Execute bonjour locally with Docker
 3. The application will be running at: <http://localhost:8080/api/bonjour>
 
 
+Run bonjour in OpenShift
```

Comment:
@helio-frota I believe that the Readme.md should remain unchanged, so this should be reverted.